### PR TITLE
VZ-9061 Backup and Restore documentation changes

### PR DESCRIPTION
**Pull request by ayaranga**
- Moved Backup and Upgrade documentation out of uninstall folder
- Fixes to examples used in backup and restore documentation
- Added missing details to recover the currently documented components on a different cluster

### CODE REVIEW

File: content/en/docs/backup/_index.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
-title: "Back Up and Restore"
+title: "Backup and Restore"
```
[the remark this answers is not in the document]
Comment (ayaranga) [reply content/en/docs/backup/_index.md:2]: I was looking at OpenShift documentation https://docs.openshift.com/container-platform/4.10/backup_and_restore/index.html for reference. They had "Backup" in the title. In the description, I see we have "back up" only. Is there any change suggested as part of this comment?

[the remark this answers is not in the document]
Comment (MelwynPaul) [reply content/en/docs/backup/_index.md:2]: Yes, let us please change the title.
Backup and Restore > Back Up and Restore

What Rosemary mentions above is from the Oracle Style Guide. Openshift might follow a different style. 
Yes, the description is correct. We just need to undo the title change you have made. 
I pasted the entire info (para) that Rosemary has mentioned in a previous PR.

[the remark this answers is not in the document]
Comment (ayaranga) [reply content/en/docs/backup/_index.md:2]: Oh ok. Got it.

File: content/en/docs/backup/opensearch.md [hunk 2]
```diff
@@ -225,48 +235,49 @@ To initiate an OpenSearch restore operation, first delete the existing OpenSearc
...
-                    - /usr/share/opensearch/bin/verrazzano-backup-hook
-                    - -operation
-                    - restore
-                    - -velero-backup-name
-                    - verrazzano-opensearch-backup
-                 waitTimeout: 30m
-                 execTimeout: 30m
-                  onError: Fail
-  EOF
+     labelSelector:
```
Comment (david-cabelus-oracle) [inline content/en/docs/backup/opensearch.md:256]: I assume that these space changes were verified? These need to be correct.

**Review by david-cabelus-oracle — APPROVED** [review]

**Review by rosemarymarano — APPROVED** [review]

File: content/en/docs/backup/opensearch.md (hunk 2 above)
Comment (ayaranga) [reply content/en/docs/backup/opensearch.md:256]: Yes, I had verified. Whatever we had earlier was not working. I will verify once again before merging.

File: content/en/docs/backup/_index.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
 ---
 title: "Back Up and Restore"
 description: "Learn how to back up and restore Verrazzano"
-weight: 2
+weight: 7
 draft: false
 ---
```
Comment (theabmitra) [inline content/en/docs/backup/_index.md:7]: This section needs to be modified totally. One of the feedbacks was this section was not clear and we should talk about same cluster and new cluster here as well .

File: content/en/docs/backup/keycloak.md [hunk 4]
```diff
@@ -229,4 +229,16 @@ kubectl scale sts -n keycloak keycloak --replicas=0
 kubectl scale sts -n keycloak keycloak --replicas=${KEYCLOAK_REPLICAS}
 kubectl wait -n keycloak --for=condition=ready pod -l app.kubernetes.io/instance=keycloak -timeout=600s
 ```
+**NOTE**:  If you are restoring the Keycloak on a different cluster, then make sure that the following secrets in the
+`verrazzano-system` namespace are updated in the new cluster with the corresponding values from the
+original cluster:
+- verrazzano
+- verrazzano-es-internal
+- verrazzano-prom-internal
+
```
[the remark this answers is not in the document]
Comment (MelwynPaul) [reply content/en/docs/backup/keycloak.md:238]: @david-cabelus-oracle 
Sure. We will do that. 👍

**Review by kminder — APPROVED** [review]
Approved.

File: content/en/docs/backup/opensearch.md (hunk 2 above)
Comment (ayaranga) [reply content/en/docs/backup/opensearch.md:256]: Verified once again. Thanks.

**Pull request merged**